listenbrainz/webserver/views/user.py

```diff
@@ -203,6 +204,18 @@ def playlists(user_name: str):
     if not current_app.config.get("FEATURE_PLAYLIST", False):
         raise NotFound()
     
+    offset = request.args.get('offset', 0)
+    try:
+        offset = int(offset)
+    except ValueError:
+        raise BadRequest("Incorrect int argument offset: %s" % request.args.get("offset"))
```

Comment:
mentioned previously: api doesn't verify data in as much detail as this. Should merge verification logic

Reply:
It does now.